setup.py

```diff
@@ -81,7 +81,13 @@ def run_process(args, **kwargs):
     try:
         run_process(("cmake", "--version"))
     except Exception as ex:
-        raise RuntimeError(_make_message_box("!! CMake not found !!")) from ex
+        raise RuntimeError(
+            _make_message_box(
+                "!! CMake not found !!",
+                "Ensure PIP is updated to the latest version to handle build dependencies automatically,",
```

Comment:
LGTM, I would consider to prefix this with "Hint" and maybe some blank lines before and after just so this part is more visibile. Often people get overwhelmed by long error messages and won't see the tips/solution.

Reply:
It will look like this: 
![image](https://github.com/user-attachments/assets/a3aaa97a-3d51-4074-869d-bae8f32201ce)

Reply:
looks good